components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -193,4 +250,85 @@ private RecoveryResponse recover(RecoveryRequest recoveryRequest, String tenantD
                 localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames,
                 localVarReturnType);
     }
+
+    private ResendResponse resend(ResendRequest resendRequest, String tenantDomain, Map<String, String> headers,
+                                  String localVarPath) throws ApiException {
+
+        // Verify if the required parameter 'recoveryRequest' is set.
+        if (resendRequest == null) {
+            throw new ApiException(HttpServletResponse.SC_BAD_REQUEST,
+                    "Missing the required parameter 'resendRequest' when requesting recovery");
+        }
+        if (StringUtils.isBlank(tenantDomain)) {
+            tenantDomain = MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
+        }
+        basePath = IdentityManagementEndpointUtil.getBasePath(tenantDomain,
+                IdentityManagementEndpointConstants.UserInfoRecovery.RECOVERY_API_V2_RELATIVE_PATH);
+        apiClient.setBasePath(basePath);
+        Map<String, String> localVarHeaderParams = new HashMap<>();
+        if (MapUtils.isNotEmpty(headers)) {
+            localVarHeaderParams.putAll(headers);
+        }
+        final String localVarAccept = apiClient.selectHeaderAccept(
+                new String[]{FrameworkConstants.ContentTypes.TYPE_APPLICATION_JSON});
+        final String localVarContentType = apiClient.selectHeaderContentType(
```

Comment:
Do we really need to define two different variables for localVarAccept and localVarContentType?

Reply:
Imo having these two in two variables makes the code clearer opposed to having this inside the parameter list of invokeAPI function call.